subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/samples/AutoTestedSamplesCoreApiIntegrationTest.groovy

```diff
@@ -20,6 +20,10 @@ import org.gradle.integtests.fixtures.AbstractAutoTestedSamplesTest
 import org.junit.Test
 
 class AutoTestedSamplesCoreApiIntegrationTest extends AbstractAutoTestedSamplesTest {
+    @Override
+    void beforeSample() {
+        executer.noDeprecationChecks()
+    }
```

Comment:
I would rather like to see the examples fixed. It's part of the deprecation imo.

If not addressed in the PR, can we create a follow up issue which we address soonish?

Reply:
It doesn't make sense to fix them because the auto-tested samples _precisely_ test calls to `force.`

Reply:
My bad, these are the Javadoc samples, right? Mixed them up with the docs samples. Sorry.

Reply:
That's right.